src-gui/src/renderer/components/modal/introduction/slides/Slide05_KeepAnEyeOnYourSwaps.tsx

```diff
@@ -0,0 +1,23 @@
+import { Link, Typography } from '@material-ui/core'
+import SlideTemplate from './SlideTemplate'
+import imagePath from '../../../../../assets/mockHistoryPage.webp'
+
+export default function Slide05_KeepAnEyeOnYourSwaps(props: slideProps) {
+    return (
+        <SlideTemplate
+            title="Monitor Your Swaps"
+            stepLabel="Step 3"
+            {...props}
+            imagePath={imagePath}
+        >
+            <Typography>
+                While the Atomic Swap Protocol is secure, you should monitor active swaps in the history tab to ensure everything proceeds smoothly.
+            </Typography>
+            <Typography>
+                <Link onClick={() => open('https://docs.unstoppableswap.net/usage/first_swap')}>
```

Comment:
This link does not work because we need to use the `open` method from `@tauri-apps/plugin-shell`. See `src-gui/src/renderer/components/icons/LinkIconButton.tsx` for an example. Might make sense to create a wrapper component for this (e.g call it `ExternalBrowserLink` / `TauriLink`)